lib/ansible/config/base.yml

```diff
@@ -159,6 +159,18 @@ AGNOSTIC_BECOME_PROMPT:
     - {key: agnostic_become_prompt, section: privilege_escalation}
   yaml: {key: privilege_escalation.agnostic_become_prompt}
   version_added: "2.5"
+BACKUP_FILE_NAME_TEMPLATE:
+  name: Tempalte to define backup file names
+  description:
+    - Defines a template that will name backup file(s) in modules using the standard backup facilities.
+    - Available variables are basename (includes extension), extension, stripname (no extension), timestamp and pid
```

Comment:
```suggestion
    - Defines a template that will name backup file(s) in modules using the standard backup facilities.
    - Available variables are basename (includes extension), extension, stripname (no extension), timestamp and process ID (PID).
```

Reply:
will add note, but `pid` is actual name of variable available